### DISCUSSION TITLE
Deepr PHP

### DISCUSSION THREAD

**Discussion by stefanak-michal** [deeprjs/deepr#9] [Show and tell]
Hi.

I like to show you library I've made to be able to use Deepr in PHP.

You can find it here:
- [Composer](https://packagist.org/packages/stefanak-michal/deepr-php)
- [Github](https://github.com/stefanak-michal/deepr-php)

**Comment by mvila** [comment]
Hi @stefanak-michal,

That's fantastic! I'll check your implementation carefully and get back to you soon.

**Comment by mvila** [comment]
Hi @stefanak-michal,

Sorry for the delay. I checked your implementation and I'd like to discuss some important points with you. If you are up for a Google Meet, please email me at hi@mvila.me.

**Comment by stefanak-michal** [comment]
New release is out https://github.com/stefanak-michal/deepr-php/releases

I barely slept. 😄

**Comment by mvila** [comment]
Wow! Well done @stefanak-michal!

There are several points I'd like to discuss. Would you be up for another Google Meet? If so, please grab a slot in my Calendly. 😄

**Comment by stefanak-michal** [comment]
I spend some time thinking and some time programming. I refactored whole process and completely removed structure classes and interfaces. I was looking into php features how to add additional meta informations to methods and properties (expose) and I realized I don't have to do it. It can be part of deserialization step.  
Both deserialization and serialization are not part of Deepr runtime.  

https://github.com/stefanak-michal/deepr-php/releases/latest

**Reply by mvila** [reply]
A million thanks, @stefanak-michal.

I listed your runtime in a new ["Community runtimes"](https://github.com/deeprjs/deepr#community-runtimes) section of Deepr README.
